### PR TITLE
PWGHF: Add XicToPKPi to taskMcEfficiency.cxx

### PR DESCRIPTION
**Pull request by phymanshu**
* Adding XicToPKPi decay channel to taskMcEfficiency.cxx

### CODE REVIEW

File: PWGHF/Tasks/taskMcEfficiency.cxx [hunk 1]
```diff
@@ -790,9 +836,19 @@ struct HfTaskMcEfficiency {
...
+
+  void processMcLcXic(soa::Join<aod::HfCand3Prong, aod::HfSelLc, aod::HfSelXicToPKPi> const& candidates,
+                      TracksWithSelectionMC const& tracks,
+                      aod::McParticles const& mcParticles,
+                      aod::McCollisionLabels const& colls)
+  {
+    std::vector<int> pdgCodes{Pdg::kLambdaCPlus, Pdg::kXiCPlus};
+    candidate3ProngMcLoop<false, false, true, true>(candidates, tracks, mcParticles, colls, pdgCodes);
+  }
+  PROCESS_SWITCH(HfTaskMcEfficiency, processMcLcXic, "Process MC for Lc and XicPlus signals ", false);
```
Comment (fcatalan92) [inline PWGHF/Tasks/taskMcEfficiency.cxx:851]: I would avoid to add these combinations, which easily grow uncontrollably (e.g. why not DsXic, DplusXic, etc.)

**Review by fcatalan92 — CHANGES_REQUESTED** [review]
Hi @phymanshu, thanks for the developments. I have just a couple of suggestions.

**Comment by phymanshu** [comment]
Hi @fcatalan92 
thanks for the reviewing it. I modified the code accordingly.

**Review by fcatalan92 — APPROVED** [review]
Thanks @phymanshu!

**Pull request merged**